QuestionBot/QuestionBot.UnitTests/Model/StoreTests.cs

```diff
@@ -1,13 +1,77 @@
-﻿using NUnit.Framework;
```

Comment:
Don't forget to delete this file now that you've changed the name. If you need help with the git command then try [this link](http://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo)